python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1320,6 +1329,11 @@ def run(self, *, ccdExposure, dnlLUT=None, bias=None, deferredChargeCalib=None,
             bfKernelOut, bfGains = self.getBrighterFatterKernel(detector, bfKernel)
             ccdExposure = self.applyBrighterFatterCorrection(ccdExposure, flat, dark, bfKernelOut, bfGains)
 
+        if self.config.doFlat:
+            # Input units: electrons
+            self.log.info("Applying flat correction.")
+            self.flatCorrection(ccdExposure, flat)
```

Comment:
This `flatCorrection` method has not been implemented yet? Will there be future tickets to do so, and to add testing? If so, perhaps we could add their numbers here as comments.

Reply:
Ah good catch - thanks. For the moment we will use the same as for the original task, which I meant to copy over.